docs/manual/developer_guide.adoc

```diff
@@ -792,12 +792,103 @@ The <tt>/var/tmp</tt> directory is a world-writable directory used for temporary
 
 A rule may contain those reference-type attributes:
 
-* `identifiers`: This is related to products/CCEs that the rule applies to; this is a dictionary, whose keys should be `cce` and a value. If `cce` is modified with a product (e.g., `cce@rhel6`), it restricts which products those identifiers applies to.
-* `references`: This is related to the compliance document line items that the rule applies to. These can be attributes such as `stig`, `srg`, `nist`, etc., whose keys may be modified with a product (e.g., `stig@rhel6`) to restrict what products an identifiers applies to.
+* `identifiers`: This is related to products that the rule applies to; this is a dictionary. Currently, only the Common Configuration Enumeration or CCE identifier is supported. Other identifiers can be added as well. Contributions to add these other identifiers are welcomed. The table below shows a list of common identifiers and their current support in a rule:
 +
-When the rule is related to RHEL, it should have a CCE.
+[cols="3*", options="header"]
+|===
+|URI |Supported |Identifier Value Description
+|http://cce.mitre.org
+|Yes
+|Common Configuration Enumeration (CCE) – the identifier value MUST be a CCE version 5 number
+
+|http://cpe.mitre.org
+|No
+|CPE –the identifier value MUST be a CPE version 2.0 or 2.3 name
+
+|http://cve.mitre.org
+|No
+|CVE –the identifier value MUST be a CVE number
+
+|http://www.cert.org
+|No
+|CERT Coordination Center – the identifier value SHOULD be a CERT advisory identifier (e.g., “CA-2004-02”)
+
+|http://www.kb.cert.org
+|No
+|US-CERT vulnerability notes database – the identifier value SHOULD be a vulnerability note number (e.g., “709220”)
+
+|http://www.us-cert.gov/cas/techalerts
+|No
+|US-CERT technical cyber security alerts –the identifier value SHOULD be a technical cyber security alert ID (e.g., “TA05-189A”)
+|===
++
+When the rule is related to RHEL, it should have a CCE. A CEE (e.g. cce@rhel7: 80328-8) is used as a global identifier that maps the rule to the product over the lifetime of a rule. There should only be one CCE mapped to a rule as a global identifier. Any other usage of CCE is no longer considered a best practice.
 Available CCEs that can be assigned to new rules are listed in the `shared/references/cce-rhel-avail.txt` file.
-See `linux_os/guide/system/software/disk_partitioning/encrypt_partitions.rule` for an example of reference-type attributes.
++
+* `references`: This is related to the compliance document line items that the rule applies to. These can be attributes such as `stig`, `srg`, `nist`, etc., whose keys may be modified with a product (e.g., `stig@rhel6`) to restrict what products a reference identifier applies to. Depending on the type of reference (e.g. catalog, rule) will depend on how many can be added to a single rule.
```

Comment:
STIG IDs (as well as CCEs) now have to be product-qualified, as we don't want Ubuntu STIG IDs.

Reply:
Not 100% sure of what you are asking here, but I add a blurb that CCEs and STIGIDs apply to certain products and product versions and cannot be applied to multiple product/versions.